### DISCUSSION TITLE
Reordering favourite repos conflict with pull to refresh gesture

### DISCUSSION THREAD

**Discussion by equeim** [community/community#122974] [Mobile] [answered]
### Select Topic Area

Bug

### Body

On the favorite repos screen in the mobile app it's not possible to drag a repository down (dragging up works) because it triggers pull to refresh instead.

**Comment by alcere** [comment, marked as answer]
Hi @equeim! We can reproduce the issue and we will work on a solution soon. Thanks for the report!
